src/BlackLinter.php

```diff
@@ -59,6 +60,10 @@ public function getLinterConfigurationOptions() {
         'type' => 'optional int',
         'help' => pht('Specify a non-default line length for black'),
       ),
+      'black.normalizestring' => array(
```

Comment:
I think we can make this more generic by having a single `black.flags` option instead that accepts an array of strings that we treat as additional flags.

And then we could have `black.options` that takes a dictionary for things like `--line-length` and `--target-version` that require a value.

Do you think that would be an improvement?

Reply:
Updated to use flags array (similar to flake8 linter)